### PR TITLE
GRO-265: Past Auctions does not display Artworks if it has no published artworks

### PR DESCRIPTION
**Pull request by The-Beez-Kneez**
This is to update Past Auctions display so that user sees text only auction title when closed auction has unpublished works. Bonhams, Ketterer, Bruun, and Swann all require us to unpublish all lots after a sale has closed.

New Behavior: 
Screen Shot 2021-04-06 at 1 24 21 PM
 
 
 JIRA: https://artsyproduct.atlassian.net/browse/GRO-265

### CODE REVIEW

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx [hunk 1]
```diff
@@ -15,6 +16,7 @@ export interface AuctionArtworksRailArtworksProps {
   sale: AuctionArtworksRailArtworks_sale
   tabType: TabType
 }
+//TODO: Need to put in conditional check to return no artwork images if the artworks is empty
```
Comment (damassi) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx:19]: Is this TODO referring to this pr?

File: src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx [hunk 2]
```diff
@@ -24,32 +26,40 @@ const AuctionArtworksRailArtworks: React.FC<AuctionArtworksRailArtworksProps> =
   const { contextPageOwnerType } = useAnalyticsContext()
   const contextModule = tabTypeToContextModuleMap[tabType]
 
+  const AUCTION_ARTWORKS_RAIL_HEIGHT = 233
+
+  if (sale.artworksConnection.edges.length === 0) {
+    return null
+  }
+
```
Comment (The-Beez-Kneez) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx:34]: This logic is what hides the Artwork images. If we do not have any artworks to display we return null.

Comment (damassi) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRail/AuctionArtworksRailArtworks.tsx:34]: Ahh I see, it hides _all_ the images -- i was thinking it just hid unpublised ones, my confusion.

**Review by damassi — APPROVED** [review]
LGTM! 👍 

One request @The-Beez-Kneez - can you add a link to the JIRA ticket in the PR description? It helps other devs when reviewing.

**Comment by The-Beez-Kneez** [comment]
@damassi how do you make it auto-connect with a ticket name? I thought it was by typing the ticket name in the title (GRO-265)

**Comment by damassi** [comment]
Unfortunately there's no way to auto link into the PR description -- what happens in the background though is we have a service (Peril) that will see the ID and link the PR to the JIRA ticket in a comment, but it doesn't work both ways. So for now we just gotta copy / paste the link directly in.

File: src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx [hunk 3]
```diff
@@ -7,33 +7,37 @@ interface FairExhibitorRailPlaceholderProps {
   done?: boolean
 }
 
+const AUCTION_ARTWORKS_RAIL_HEIGHT = 233
```
Comment (jonallured) [inline src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx:10]: **suggestion [non-blocking]:** extract hardcoded heights to exported object

This knowledge is spreading, maybe defining a const of heights would be helpful:

```ts
// maybe in a helpers file?
export const auctionHeights = {
  artworksRail: 233,
  artworksImage: 160
}

// then later:
import { auctionHeights } from "./some/helpers/file.ts"
<Box height={auctionHeights.artworksRail} />
```

*format via [Conventional Comments](https://conventionalcomments.org/)*

**Review by jonallured — COMMENTED** [review]
Nice one!! Just a suggestion about the hardcoded heights.

File: src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx (hunk 3 above)
Comment (The-Beez-Kneez) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx]: Updated per suggestion. I extracted the values out to a Helper file and updated all usages of it.

Comment (damassi) [reply src/v2/Apps/Auctions/Components/AuctionArtworksRailPlaceholder.tsx]: Much better with the shared constant! I made one lil fix in my pr tho -- we already had a `Utils` directory in the app folder so moved this there, from Helpers (we generally use `Utils` elsewhere).

**Pull request merged**